tools/webhook-apicoverage/resourcetree/README.md

```diff
@@ -0,0 +1,12 @@
+# Resource Tree
+
+resourcetree package contains types and interfaces that defines resource tree- the n-arry tree based representation of an API resource. This representation uses the concept of forrest to group together all resource trees that are part of the entire surface area. As an example for Knative Serving API we would have individual resource trees for Configuration, Revision, Route and Service which are encapsulated inside a resource forest under version v1alpha1.
```

Comment:
can we use something more specific instead of "entire surface area"? Might be hard for reader to understand. 
Also it's better to bring up forest at the end of paragraph once resourcetree is explained fully.

Reply:
Removed the ambiguity. I have also moved the resourceForest lines into its own section.